src/app/pages/server-compare/server-compare.component.html

```diff
@@ -43,7 +43,7 @@ <h1 class="text-white font-bold text-3xl">Server Comparison</h1>
         <table class="items_table" [style]="getMainTableWidth()">
           <thead>
             <tr>
-              <th style="max-width: 250px; width: 250px">
+              <th style="max-width: 250px; width: 250px; min-width: 250px;">
```

Comment:
I'm not sure if enforcing a fixed 250px width with min/max/explicit width is the best option here .. cannot we try the solution mentioned in the ticket? e.g. adding `white-space: nowrap;` to the related `td` elements

Reply:
This is why:
![image](https://github.com/user-attachments/assets/510c0f07-5524-40bd-9ce7-f72530980424)
We have to know the width of the first column, because of the sticky header. As seen in the picture the sticky header has different width than the one with the data.